lib/mod.ts

```diff
@@ -156,16 +156,18 @@ export async function findPullRequests(options: FindPullRequestOptions) {
   const resp = await octono.request("GET /repos/{owner}/{repo}/pulls", {
     owner: owner,
     repo: repo,
-    labels: label,
     state: state,
     headers: {
       authorization: `bearer ${await fetchGitHubToken()}`,
     },
   });
 
-  const items = await resp.json();
-
-  return items;
+  const items = await resp.json()
+  
+  return items.filter(pr => {
```

Comment:
I think this can be improved by using the `pr.labels.map(label => name).includes(label)` method, keeping the clarity. What do you think?

Reply:
totally, got this temp solution while coding but forgot to inline it, I will do the change